translate/tools/pocount.py

```diff
@@ -46,6 +46,34 @@
 default_style = style_full
 
 
+class bcolors:
+    """Class to implement color mode
+    """
+    @staticmethod
+    def HEADER():
+        return "\033[95m" if bcolors.color_mode else ""
+
+    @staticmethod
+    def OKGREEN():
+        return "\033[92m" if bcolors.color_mode else ""
+
+    @staticmethod
+    def WARNING():
+        return "\033[93m" if bcolors.color_mode else ""
+
+    @staticmethod
+    def FAIL():
+        return "\033[91m" if bcolors.color_mode else ""
+
+    @staticmethod
+    def ENDC():
+        return "\033[0m" if bcolors.color_mode else ""
+
+
+# print using color? Default to true
+bcolors.color_mode = True
```

Comment:
set this attribute inside the class, which helps static analysers know that `color_mode` is an attribute while lexing the static methods which use it.